sdcm/cluster.py

```diff
@@ -5124,12 +5124,13 @@ def node_setup(self, node, **kwargs):  # pylint: disable=unused-argument
         self.mgmt_auth_token = self.monitor_id  # pylint: disable=attribute-defined-outside-init
 
         if self.test_config.REUSE_CLUSTER:
-            node.disable_daily_triggered_services()
+
             self.configure_scylla_monitoring(node)
             self.restart_scylla_monitoring(sct_metrics=True)
             set_grafana_url(f"http://{normalize_ipv6_url(node.external_address)}:{self.grafana_port}")
             return
 
+        node.disable_daily_triggered_services()
```

Comment:
This change doesn't fit the `mocking` part. It is change of logic for other backends.
If it is intentional then we have bug now? Or why is it changed?

Reply:
now on it's own commit